app/controllers/AccountManagement.scala

```diff
@@ -23,14 +26,21 @@ import scalaz.std.scalaFuture._
 import scalaz.syntax.std.option._
 import scalaz.{EitherT, OptionT, \/}
 
-object AccountManagement extends Controller with LazyLogging {
-
-  val accountManagementAction = NoCacheAction
+// this handles putting subscriptions in and out of the session
```

Comment:
I split this file into a few objects, one to handle storing retrieving subscriptions in the session, one for weekly manage stuff, and one for delivery manage stuff.
The implicits got a bit out of hand though, as everything needs rather more than I'd hope.